src/Tizen.Applications.Alarm/Tizen.Applications/AlarmManager.cs

```diff
@@ -74,6 +74,33 @@ public enum AlarmWeekFlag
         WeekDays = Monday | Tuesday | Wednesday | Thursday | Friday
     }
 
+    /// <summary>
+    /// Enumeration for inexact interval.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public enum AlarmStandardPeriod
```

Comment:
I don't know the purpose of this enum, but based on the discussion with the reviewer of native API in C# you cannot freely pass enum values as integers, because it would require the explicit conversion to int. Maybe using the constant values is the proper solution in this case.

Reply:
I added new overloading method for AlarmStandardPeriod